### PR TITLE
Add SKIP_MOTD to profile as a suppression valve

### PR DESCRIPTION
**Pull request by tianon**
See https://groups.google.com/d/topic/msysgit/P7a2MVF0ok8/discussion for the context (which never led to a PR).

### CODE REVIEW

File: etc/profile [hunk 1]
```diff
@@ -127,7 +127,7 @@ test -x /share/msysGit/post-checkout-hook &&
 mkdir -p /.git/hooks &&
 cp /share/msysGit/post-checkout-hook /.git/hooks/post-checkout
 
-test -f /etc/motd && sed "s/\$MESSAGE/$MESSAGE/" < /etc/motd
+test -z "$SKIP_MOTD" && test -f /etc/motd && sed "s/\$MESSAGE/$MESSAGE/" < /etc/motd
```
Comment (tianon) [inline etc/profile]: Would this be better as `test -z "$SKIP_MOTD" -a -f /etc/motd ...`?

**Comment by tianon** [comment]
Of course, happy to rebase, amend, etc as necessary/requested. :+1:

File: etc/profile (hunk 1 above)
Comment (dscho) [inline etc/profile]: Good idea!

**Comment by ahmetalpbalkan** [comment]
:+1: for this.

File: etc/profile (hunk 1 above)
Comment (tianon) [inline etc/profile]: :+1: updated from my phone :smile:

**Comment by dscho** [comment]
Thanks, @tianon.

**Comment by tianon** [comment]
My pleasure! :+1:

**Pull request merged**